sdk/servicebus/azure-servicebus/azure/servicebus/aio/_servicebus_receiver_async.py

```diff
@@ -160,27 +185,34 @@ def _create_handler(self, auth):
             encoding=self._config.encoding,
             receive_settle_mode=self._mode.value,
             send_settle_mode=SenderSettleMode.Settled if self._mode == ReceiveSettleMode.ReceiveAndDelete else None,
-            timeout=self._idle_timeout * 1000 if self._idle_timeout else 0,
-            prefetch=self._prefetch
+            timeout=self._max_wait_time * 1000 if self._max_wait_time else 0,
+            prefetch=self._prefetch,
+            shutdown_after_timeout=False
         )
 
     async def _open(self):
         if self._running:
             return
-        if self._handler:
+        if self._handler and self._handler._shutdown:
```

Comment:
in sync code, it's `if self._handler and not self._handler._shutdown:`
missing "not" here